### PR TITLE
[SPARK-14339][DOC] Add python examples for DCT,MinMaxScaler,MaxAbsScaler

### PR DESCRIPTION
**Pull request by zhengruifeng**
## What changes were proposed in this pull request?

add three python examples
## How was this patch tested?

manual tests

### CODE REVIEW

File: examples/src/main/python/ml/dct_example.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+if __name__ == "__main__":
+    sc = SparkContext(appName="DCTExample")
+    sqlContext = SQLContext(sc)
+
+    # $example on$
+    df = sqlContext\
+        .createDataFrame([(Vectors.dense([0.0, 1.0, -2.0, 3.0]),),
+                          (Vectors.dense([-1.0, 2.0, 4.0, -7.0]),),
+                          (Vectors.dense([14.0, -2.0, -5.0, 1.0]),)],
+                         ["features"])
```
Comment (yanboliang) [inline examples/src/main/python/ml/dct_example.py]: Better code format, you can refer [here](https://github.com/apache/spark/blob/master/examples/src/main/python/ml/aft_survival_regression.py#L32).

**Comment by yanboliang** [comment]
LGTM except minor issue.

File: examples/src/main/python/ml/dct_example.py (hunk 1 above)
Comment (zhengruifeng) [inline examples/src/main/python/ml/dct_example.py]: thanks, I will fix it

**Comment by mengxr** [comment]
@zhengruifeng Could you make JIRAs for this PR and #11844? `[MINOR]` is only used for apparent typo fixes or trivial changes.

**Comment by zhengruifeng** [comment]
@mengxr Ok, I will open the missing JIRAs

**Comment by yanboliang** [comment]
LGTM

**Comment by zhengruifeng** [comment]
cc @jkbradley

**Comment by mengxr** [comment]
Merged into master. Thanks!

**Pull request closed without merging**